torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -13,15 +13,64 @@ namespace {
 // Some of these restrictions may be relaxable, but you should
 // carefully read the code first, as we rely on these assumptions.
 std::unordered_set<NodeKind> simple_mappable = {
-  ksigmoid,
-  ktanh,
-  kmul,
+  k__and__,
+  k__lshift__,
+  k__or__,
+  k__rshift__,
+  k__xor__,
+  kabs,
+  kacos,
   kadd,
+  kasin,
+  katan,
+  katan2,
+  kceil,
+  kclamp,
+  kcos,
+  kcosh,
+  kdiv,
+  keq,
+  kexp,
+  kfloor,
+  kfmod,
+  kfrac,
+  kge,
+  kgt,
+  kle,
+  klerp,
+  klgamma,
+  klog,
+  klog1p,
+  klt,
+  kmax,
+  kmin,
+  kmul,
+  kne,
   kneg,
+  kones,
+  kpow,
+  kreciprocal,
+  kremainder,
+  kround,
+  krsqrt,
+  ksigmoid,
+  ksin,
+  ksinh,
+  ksqrt,
+  ksub,
+  ktan,
+  ktanh,
+  ktrunc,
+  kzeros,
```

Comment:
maybe instead of maintaining this huge list of interned strings it would just be easier to do this:
```cpp
std::unordered_set<NodeKind> simple_mappable = fmap(std::vector<std::string>{
  "trunc",
  "sigmoid",
  "atan",
  ...
}, stringToSymbol);
```

Most of them aren't used anywhere outside the lists here and in `graph_fuser.cpp`, which could do the same trick

Reply:
I think we should codegen interned strings for every identifier mentioned in ATen. But not in this PR.